libs/client/pipes/src/lib/pipes/display-tags.pipe.ts

```diff
@@ -0,0 +1,29 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Genres, TagsModel } from '@dragonfish/shared/models/content';
+
+@Pipe({ name: 'displayTags' })
+export class DisplayTagsPipe implements PipeTransform {
+    transform(tags: TagsModel | TagsModel[]) {
+        if (!Array.isArray(tags)) {
+            return this.formatTagName(tags);
+        }
+        if (tags.length === 1) {
+            return this.formatTagName(tags[0]);
+        } else {
+            const theseTags: string[] = [];
+            tags.forEach((tag) => {
+                theseTags.push(this.formatTagName(tag));
+            });
+            return theseTags.join(', ');
```

Comment:
If you want, you should be able to avoid the intermediate variable with `.map()`:
```suggestion
            return tags.map(x => this.formatTagName(x)).join(', ');
```